### PR TITLE
Add the fourth post. (#12)

### PR DESCRIPTION
**Pull request by andrewtar**

### CODE REVIEW

File: _i18n/ru/_posts/2017-10-31-generative-modeling-with-deep-learning.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+## Другие задачи
+
+В дополнение к изученным выше задачам классификации и регрессии есть и другие задачи так называемого обучения с учителем, в основном сводящиеся к отображению между точками и последовательностями: Object-to-Sequence, Sequence-to-Sequence, Sequence-to-Object. Так же есть и большой спектр классических задач обучения без учителя: кластеризация, заполнение пробелов в данных, и, наконец, явная или неявная аппроксимация распределений, которая и используется для генеративного моделирования. Именно о последнем классе задач будет идти речь в этом цикле статей.
+
+## Генеративные модели
+
+В следующей главе мы посмотрим, что такое генеративные модели и чем они принципиально отличаются от рассмотренных в этой главе дискриминативных. Мы посмотрим на простейшие примеры генеративных моделей и попробуем обучить модель, генерирующую сэмплы из простого распределения данных.
+
+## Благодарности
+- Спасибо [Olga Talanova](https://www.linkedin.com/in/olga-talanova-b319b761/) за ревью текста.
```
Comment (Monnoroch) [inline _i18n/ru/_posts/2017-10-31-generative-modeling-with-deep-learning.md]: Не надо пунктов. Просто два предложения.

File: _i18n/en/_posts/2017-10-31-generative-modeling-with-deep-learning.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (Monnoroch) [inline _i18n/en/_posts/2017-10-31-generative-modeling-with-deep-learning.md]: > Required statuses must pass before merging

File: _i18n/ru/_posts/2017-10-31-generative-modeling-with-deep-learning.md (hunk 1 above)
Comment (andrewtar) [reply _i18n/ru/_posts/2017-10-31-generative-modeling-with-deep-learning.md]: Done.

**Comment by Monnoroch** [comment]
Related to #12.

**Pull request merged**